crates/generate/src/types/struct.rs

```diff
@@ -93,6 +118,9 @@ pub(super) fn define_struct(
             }
 
             fn read(location: &wiggle_runtime::GuestPtr<'a, Self>) -> Result<Self, wiggle_runtime::GuestError> {
+                let _ =
+                    location.mem()
+                        .validate_size_align(location.offset(), Self::guest_align(), Self::guest_size())?;
```

Comment:
I thought that the member reads themselves would validate this, but was wondering if I'm missing something subtle why there's a double-check here?

Reply:
correct, they will. will fix